File: 1100/1708B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() 
{
    int t;
    cin >> t;

    while (t--) 
    {
        int n;
        long long l, r;
        cin >> n >> l >> r;

        vector<long long> a(n);
        bool flag = true;

        for (int i = 1; i <= n; i++) 
        {
            if (l % i == 0)
                a[i - 1] = l;
            else 
            {
                long long val = (l / i + 1) * i;

                if (val > r) 
                {
                    flag = false;
                    break;
                }

                a[i - 1] = val;
            }
        }

        if (!flag)
            cout << "NO\n";
        else 
        {
            cout << "YES\n";
            for (int i = 0; i < n; i++)
                cout << a[i] << " ";
            
            cout << "\n";
        }
    }

    return 0;
}

File: 1100/1708B/main.java
import java.util.*;
public class main
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        
        while(t-- > 0)
        {
            int n = sc.nextInt();
            long l = sc.nextLong();
            long r = sc.nextLong();
            
            long a[] = new long[n];
            boolean flag = true;
            
            for(int i=1;i<=n;i++)
            {
                if(l%i == 0)
                    a[i-1] = l;
                else
                {
                    long val = (l/i + 1)*i;
                    if(val > r)
                    {
                        flag = false;
                        break;
                    }
                    
                    a[i-1] = val;
                }
            }
            
            if(flag == false)
                System.out.println("NO");
            else
            {
                System.out.println("YES");
                for(int i=0;i<n;i++)
                    System.out.print(a[i] + " ");
                System.out.println();
            }
        }
    }
}
